msg/ServoReport.msg

```diff
@@ -0,0 +1,9 @@
+uint64 timestamp					# time since system start (microseconds)
+
+float32 servo_position					# feedback position of servo. [-1;1]
+float32 servo_speed					# feedback speed of servo [deg/s]
+float32 servo_acceleration				# feedback of acceleration
```

Comment:
Just curious, what would you use acceleration feedback for?

Reply:
We are not using it for something specific currently, but since the servo provides feedback about its acceleration, we  should at least log it.